tests/js/spec/utils/replays/getCurrentUrl.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import getCurrentUrl from 'sentry/utils/replays/getCurrentUrl';
+import ReplayReader from 'sentry/utils/replays/replayReader';
+
+import {
+  createEventTransaction,
+  createReplayEvents,
+  createRrWebEvents,
+} from './mocks/events.mock';
+
+const CURRENT_TIME_MS = 55114;
+
+function createReplay(): ReplayReader | null {
+  return ReplayReader.factory(
+    createEventTransaction(),
+    createRrWebEvents(),
+    createReplayEvents()
+  );
+}
+
+describe('getCurrentUrl', () => {
+  it('should return the current url given a replay', () => {
+    const replay = createReplay() as ReplayReader;
+    expect(replay).toBeDefined();
+    expect(getCurrentUrl(replay, 0)).toBe('https://sourcemaps.io/');
+  });
+
+  it('should return a different current url if Navigation action happens', () => {
```

Comment:
This doesn't really reflect what we're testing, there isn't a "navigation action" happening in the test